x-pack/plugins/alerting/server/saved_objects/schemas/raw_rule/v1.ts

```diff
@@ -88,24 +94,26 @@ const rRuleSchema = schema.object({
       schema.literal('SU'),
     ])
   ),
-  byweekday: schema.maybe(schema.arrayOf(schema.oneOf([schema.string(), schema.number()]))),
-  bymonth: schema.maybe(schema.number()),
-  bysetpos: schema.maybe(schema.number()),
-  bymonthday: schema.maybe(schema.number()),
-  byyearday: schema.maybe(schema.number()),
-  byweekno: schema.maybe(schema.number()),
-  byhour: schema.maybe(schema.number()),
-  byminute: schema.maybe(schema.number()),
-  bysecond: schema.maybe(schema.number()),
+  byweekday: schema.maybe(
+    schema.nullable(schema.arrayOf(schema.oneOf([schema.string(), schema.number()])))
+  ),
+  bymonth: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
+  bysetpos: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
+  bymonthday: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
+  byyearday: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
+  byweekno: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
+  byhour: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
+  byminute: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
+  bysecond: schema.maybe(schema.nullable(schema.arrayOf(schema.number()))),
```

Comment:
Same as above, we should either remove `| null` from the provider or make all these nullable.

Reply:
Perhaps we should go ahead and create a meta issue - RawRule v3 - then we can keep a list of the things we'd want to change in v3.